README.md

```diff
@@ -1,10 +1,22 @@
 ### Metabolic Atlas Map Viewer 3D
 
-This is a 3D graph viewer to visualize the metabolic atlas compound graph (metabolic network). See the [wiki](../../wiki) for more details.
+This is a 3D graph viewer to visualize the metabolic atlas compound graph (metabolic network), which is used as a self-contained feature of [Metabolic Atlas](https://metabolicatlas.org). The purpose of this library is to delvier the right technical solution that allows Metabolic Atlas to provide users with a useful experience while browsing and modifying a very large graph, with minimal knowledge required. The solution should be easy to work with and easy to maintain by programmers. The expectation is that most of the users of Metabolic Atlas will be biologists, followed by bioinformaticians; this means the 3D viewer should be minimalistic and intuitive. 
 
-Install dependencies with `npm install`, then run `npm run dev` to start the
-development server, or `npm run prepare` to build and minify the
-current version of the viewer.
+Below is a generic metabolic network example. Other graph structures are possible: no genes, duplicated metabolites.
```

Comment:
I feel like I'm missing something here. Is the example just an example, or does it actually put some restrains on which components that can be connected to each other?

Reply:
A lot can be said about metabolic networks, so I added a link to Wiki instead. Let me know if that is in line with your expectations.